tools/devops/automation/scripts/GitHub.psm1

```diff
@@ -438,7 +439,12 @@ class GitHubComments {
         # footer
         $this.WriteCommentFooter($msg, $commentId)
 
-        return $this.NewComment($msg)
+        $result = $this.NewComment($msg)
+        
+        # If this commit is not the latest in the PR, hide this comment immediately
+        $this.HandleNewCommentHiding($result, $commentId)
+
+        return $result
```

Comment:
Look that you do this pattern a lot. You call the method with the '$result' variable and then return $result. Can you unify it in a way that you don't have to do always the same lines all over and over?